scene/gui/popup_menu.cpp

```diff
@@ -903,7 +903,10 @@ void PopupMenu::activate_item(int p_item) {
 		next = next->get_parent();
 		pop = next->cast_to<PopupMenu>();
 	}
-	hide();
+	// Hides popup by default; unless otherwise specified using set_hide_on_item_selection(p_bool)
+	if (hide_on_item_selection) {
+		hide();
+	}
```

Comment:
`PopupMenu` allows creating submenus by adding `PopupMenu` children to the `PopupMenu`  and calling `add_submenu_item`.

When clicking a submenu item with `hide_on_item_selection` turned off, the parents will be hidden, while the submenu remains shown (see logic below the `//hide all parent PopupMenue's` comment). The parents should remain visible as well.